images/nginx/rootfs/patches/nginx-1.21.4-http2.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/src/http/v2/ngx_http_v2.c b/src/http/v2/ngx_http_v2.c
```

Comment:
Can you please add a comment of what's the origin of the patch? Also, please sign the CLA!

Thanks for the PR!!

Reply:
Source information added + CLA signed!

Reply:
Thanks!